integration/default_test.go

```diff
@@ -87,7 +87,7 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 				MatchRegexp(fmt.Sprintf(`    PYTHONPATH -> "\/layers\/%s\/poetry\/lib\/python\d+\.\d+\/site-packages:\$PYTHONPATH"`, strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))),
 			))
 			Expect(logs).To(ContainLines(
-				MatchRegexp(`Paketo CPython Buildpack \d+\.\d+\.\d+`),
+				MatchRegexp(`\s*CPython Buildpack v?\d+\.\d+\.\d+`),
```

Comment:
I can also get rid of this line if that's better

Reply:
yeah honestly I'd probably just remove it. I've been trying to remove coupling like this wherever I see it.